mm2src/mm2_main/src/rpc/dispatcher/dispatcher.rs

```diff
@@ -185,6 +184,9 @@ async fn dispatcher_v2(request: MmRpcRequest, ctx: MmArc) -> DispatcherResult<Re
         "remove_delegation" => handle_mmrpc(ctx, request, remove_delegation).await,
         "remove_node_from_version_stat" => handle_mmrpc(ctx, request, remove_node_from_version_stat).await,
         "sign_message" => handle_mmrpc(ctx, request, sign_message).await,
+        "sign_raw_transaction" => handle_mmrpc(ctx, request, sign_raw_transaction).await,
+        #[cfg(not(target_arch = "wasm32"))]
+        "sign_eth_transaction" => handle_mmrpc(ctx, request, sign_eth_transaction).await,
```

Comment:
Why does this work for non-wasm environments only? I guess it will be removed anyways after the other comments are resolved :)

Reply:
As I understand we use Metamask in the wasm environment and it looks like it does not support the sign transaction method directly (https://stackoverflow.com/questions/71923481/sign-transaction-without-broadcasting-it-with-metamask-not-using-private-key). 
But maybe this could be implemented in another way?

Reply:
We don't need to implement `sign_raw_transaction` RPC for metamask for now as we discussed in DM.